src/eduid_idp/authn.py

```diff
@@ -152,10 +152,10 @@ def __init__(self, logger, config, userdb, auth_client=None, authn_store=None):
         self.userdb = userdb
         self.auth_client = auth_client
         if self.auth_client is None:
-            self.auth_client = get_vccs_client(config.vccs_url)
+            self.auth_client = get_vccs_client(config.get('VCCS_URL'))
```

Comment:
If VCCS_URL is None, this will eventually default to http://localhost:8550/. I would suggest we move that default from VCCSClient.__init__() to the configuration defaults, and then skip the get() here. Oh, I see this default is already in the configuration defaults, so skip the get().